<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButtonStartTest">
        <property name="text">
         <string>Start loopback test</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonPrintStatistics">
        <property name="text">
         <string>Print statistics</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonEndTest">
        <property name="text">
         <string>End test</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonScanPorts">
        <property name="text">
         <string>Scan ports</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonClearLog">
        <property name="text">
         <string>Clear log</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxAvailablePorts"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClearLog</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
